.devcontainer/post_create.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+sudo apt-get update
+sudo apt-get install -y libaspell-dev
```

Comment:
Does `libaspell-dev` install `aspell-en` automatically? Is `aspell-en` installed by default in image `mcr.microsoft.com/devcontainers/python:1-3-bullseye`?

Reply:
That is correct. Here's the full list of what this command installs:
- aspell
- aspell-en
- dictionaries-common
- emacsen-common
- libaspell-dev
- libaspell15
- libtext-iconv-perl

This image won't be published anywhere, so if we wanted, we could likely skip most of these recommended dependencies and save some space.

Reply:
I was just curious, we need `aspell` to run all tests locally.